lib/records/fhir_dstu2.rb

```diff
@@ -332,6 +353,29 @@ def self.observation(observation, fhir_record, patient, encounter)
         fhir_record.entry << entry
       end
 
+      def self.quality_of_life_observation(observation, fhir_record, patient)
```

Comment:
Do you mean calling the FhirRecord method from fhir_dstu2? The code is the same, but `entry`, `entry.resource`, and `entry.resource.valueQuantity` are all `FHIR::DSTU2` types as oppose to just `FHIR`.